var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -36,7 +36,9 @@ class Acts(CMakePackage, CudaPackage):
     tags = ['hep']
 
     # Supported Acts versions
-    version('master', branch='master')
+    version('main', branch='main')
+    version('master', branch='main')  # Alias kept for script compatibility
```

Comment:
Would it work if you add `deprecated=True`? That has the benefit of not showing the alias in `spack info`

Reply:
I wouldn't mind myself as I migrated all my scripts to `main`, just kept the `master` alias for the benefit of not suddenly breaking other people's scripts.

Reply:
Sure, with `deprecated=True` scripts would not be broken but people looking at the available versions won't see `master`.

Reply:
Thanks for the clarification, I implemented the suggested change :)